SwiftSampleProject/SwiftyVokoderTests/SwiftyVokoderTests.swift

```diff
@@ -12,8 +12,39 @@ import Vokoder
 
 let grandMilwaukeeIdentifier = 30096
 
+struct CTAData {
+
+    static func allStopDictionaries() -> [[String: AnyObject]] {
+        guard let
+            path = NSBundle.mainBundle().pathForResource("CTA_stations", ofType: "json"),
+            data = NSData(contentsOfFile: path)
+            else {
```

Comment:
Per coding standards, the `else` should be on the same line as the last condition